### PR TITLE
preview5 coherent package versions for breaking ef change

### PR DESCRIPTION
**Pull request by seancpeters**
Update the packages to build scaffolding to 3.0.0-preview5 versions.
Minor fixes for breaking changes in EF.

### CODE REVIEW

File: src/VS.Web.CG.EFCore/NavigationMetadata.cs [hunk 1]
```diff
@@ -21,8 +21,9 @@ public NavigationMetadata(INavigation navigation, Type dbContextType)
             Contract.Assert(navigation.IsDependentToPrincipal());
 
             AssociationPropertyName = navigation.Name;
-
+#pragma warning disable EF1001 // Internal EF Core API usage.
             var otherEntityType = navigation.ForeignKey.ResolveOtherEntityType(navigation.DeclaringEntityType);
```
Comment (ajcvickers) [inline src/VS.Web.CG.EFCore/NavigationMetadata.cs]: @AndriySvyryd Is there a good alternative to using this internal API?

File: test/VS.Web.CG.EFCore.Test/TestModels/TestModel.cs [hunk 2]
```diff
@@ -19,7 +19,9 @@ public static IModel CategoryProductModel
         {
             get
             {
+#pragma warning disable EF1001 // Internal EF Core API usage.
                 var builder = new ModelBuilder(SqlServerConventionSetBuilder.Build());
```
Comment (ajcvickers) [inline test/VS.Web.CG.EFCore.Test/TestModels/TestModel.cs]: What is the EF internal code that is being used here? As far as I can tell, everything here is now public.

Comment (seancpeters) [reply test/VS.Web.CG.EFCore.Test/TestModels/TestModel.cs]: SqlServerConventionSetBuilder.Build() is marked as [EntityFrameworkInternal]

Comment (ajcvickers) [reply test/VS.Web.CG.EFCore.Test/TestModels/TestModel.cs]: Ah, found the issue. These are public in master, but that that hasn't propagated to the release yet. So you should be able to remove these pragmas before RTM.

File: src/VS.Web.CG.EFCore/NavigationMetadata.cs (hunk 1 above)
Comment (AndriySvyryd) [reply src/VS.Web.CG.EFCore/NavigationMetadata.cs]: No, I'll make it public in the next preview.

Comment (AndriySvyryd) [reply src/VS.Web.CG.EFCore/NavigationMetadata.cs]: Actually, in this case `navigation.GetTargetType()` can be used instead.

**Pull request merged**